### ISSUE TITLE
Test with different browsers and screen resolutions

### ISSUE THREAD

**Issue by nesnoj** [rl-institut/WAM_APP_stemp_abw#21]
Tool should be tested with different browsers including outdated versions. Based upon the findings, we should make a minimum version compulsory for each browser.

Pay special attention to the loading problems of highcharts which were solved by setting a timeout, cf.
#17
#18
#19

**Comment by 4lm** [comment]
Yes, good idea, we then also could/should check for the height of the Highcharts (HC) in the popups, which may break under certain conditions, as you also mentioned in https://github.com/rl-institut/WAM_APP_stemp_abw/pull/19#pullrequestreview-214091088. We additionally could check it by also simulating devices with different pixel densities.

**Comment by nesnoj** [comment]
This task is crucial, as (at least for 1 participant) the loading of layers didn't work with IE :(

**Comment by nesnoj** [comment]
Also, take screen resolution into account and define minimum (in WS some users had problems here).
Raise warning on landing page if browser version too old or resolution too small.

**Comment by nesnoj** [comment]
Works pretty well mit chromium engine (Chrome, Edge, ...) and only small drawbacks with FF (e.g. footer not collapsable).
But it would be good to have the warning message mentioned above. Will move to nice-to-have milestone.
